apple/internal/partials/support/resources_support.bzl

```diff
@@ -211,8 +211,8 @@ def _asset_catalogs(
         output_plist = assets_plist,
         platform_prerequisites = platform_prerequisites,
         product_type = product_type,
-        resolved_alticonstool = apple_mac_toolchain_info.resolved_alticonstool,
-        resolved_xctoolrunner = apple_mac_toolchain_info.resolved_xctoolrunner,
+        alticonstool = apple_mac_toolchain_info.alticonstool,
```

Comment:
Same alphabetical comment. I won't comment this again, just a reminder it's across multiple files.